ui/lib/config-ui/addon/controllers/messages/create.js

```diff
@@ -4,8 +4,14 @@
  */
 
 import Controller from '@ember/controller';
-export default class MessagesController extends Controller {
+import { action } from '@ember/object';
+export default class MessagesCreateController extends Controller {
   queryParams = ['authenticated'];
 
   authenticated = true;
+
+  @action
+  onUpdateEndTime(endTime) {
```

Comment:
The component should handle updating the model, I don't think we need this here?

Reply:
Followed up on slack. For posterity, args are read-only so we'll need to eventually move away from setting things on the arg argument and do it at the parent level. Since this is our pattern to set things using args, we'll continue doing that do a global change. https://github.com/hashicorp/vault/pull/24603#discussion_r1434311334